pkg/kubelet/kubelet_pods.go

```diff
@@ -1650,6 +1651,11 @@ func (kl *Kubelet) cleanupOrphanedPodCgroups(cgroupPods map[types.UID]cm.CgroupN
 	}
 	pcm := kl.containerManager.NewPodContainerManager()
 
+	// Do not delete immediately cgroups for a pod when it is first seen but leave
+	// a reasonable amount of time to the runtime to complete the cleanup.
+	delayTime := time.Second * 600
```

Comment:
This bounds deletion of a pod to 10m?

Reply:
no, this is only for the kubelet cleanup (if needed).  The runtime will still destroy the pod as soon as it is requested